src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -149,7 +149,7 @@ void getChillerASHRAE205Input(EnergyPlusData &state)
         thisChiller.Name = UtilityRoutines::MakeUPPERCase(thisObjectName);
         ip->markObjectAsUsed(state.dataIPShortCut->cCurrentModuleObject, thisObjectName);
 
-        std::string const rep_file_name = std::move(ip->getAlphaFieldValue(fields, objectSchemaProps, "representation_file_name"));
+        std::string const rep_file_name = ip->getAlphaFieldValue(fields, objectSchemaProps, "representation_file_name");
```

Comment:
Simply eliminates the std::move attempt to clean up the build warnings on Clang: https://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-f96d2dae239d0ca6b69b2c8d065980028cf2b986-x86_64-MacOS-10.17-clang-13.0.0.html

Reply:
Agreed.

Reply:
I updated the Wiki [page](https://github.com/NREL/EnergyPlus/wiki/Move-constructors-%28i.e.%2C-std%3A%3Amove%29%3A-When-and-How-to-Use-Them) to reflect the new (and by "new" I mean "new to me") guidance on the non-use of `std::move` on return values.